pkg/proxy/ipvs/proxier.go

```diff
@@ -82,6 +82,15 @@ const (
 	DefaultDummyDevice = "kube-ipvs0"
 )
 
+// ipvsTableChains is the iptables chains ipvs proxy mode used.
+var ipvsTableChains = []struct {
```

Comment:
Sorry to be a pain - this name does not convey what it means.  Something like `tableChainsWithJumpServices` (the original name) is much better

Reply:
Thanks for patient guidance.